resource_work_time_from_contracts/models/resource_mixin.py

```diff
@@ -16,8 +16,7 @@ class ResourceMixin(models.AbstractModel):
 
     _inherit = "resource.mixin"
 
-    # make this field read-only.
-    resource_calendar_id = fields.Many2one("resource.calendar", readonly=True)
+    resource_calendar_id = fields.Many2one("resource.calendar")
```

Comment:
this line should just be removed from here, as it changes nothing anymore.

Reply:
forgot to use my brain on this one : it's fixed